tests/cli_spec.rb

```diff
@@ -4,25 +4,35 @@
 
 RSpec.describe 'run tests', type: :aruba do
   it 'runs a single spec' do
-    run_command "#{Dir.pwd}/sass-spec.rb --command "\
-                "'#{Dir.pwd}/tests/sass_stub' "\
-                "#{Dir.pwd}/tests/fixtures/basic"
+    run_command command('basic')
```

Comment:
Why not something like `run_sass 'basic'` which includes the `run_command` invocation?

Reply:
That's a much, much better way of doing this.